econml/orf/_ortho_forest.py

```diff
@@ -928,7 +930,7 @@ def __init__(self, *,
         # Define parameter estimators
         parameter_estimator = DROrthoForest.parameter_estimator_func
```

Comment:
We have to move this logic to fit in this PR

Reply:
I have moved all the categories related logic to fit, but for example this one, it's also the input of super class, can't be moved to fit unless we change of the input of super class as well.